tests/integration/README.md

```diff
@@ -0,0 +1,33 @@
+## Running Integration Tests with `aws-k8s-tester`
+
+[`aws-k8s-tester`](https://github.com/aws/aws-k8s-tester) is an e2e test deployment interface for testing K8s clusters on AWS and is used in conjunction with `aws-ebs-csi-driver` for csi integration tests.
+
+### Requirements 
+
+1. macOS or Linux
+1. [Go](https://golang.org/doc/install), version 1.11+
```

Comment:
No need to mention this since it shares the same Go version as https://github.com/kubernetes-sigs/aws-ebs-csi-driver#requirements.

Reply:
done!